docs/core/diagnostics/observability-with-otel.md

```diff
@@ -0,0 +1,429 @@
+---
+title: .NET Observability with OpenTelemetry
+description: An introduction to observing .NET apps with OpenTelemetry
+ms.date: 6/14/2023
+ms.topic: conceptual
+---
+
+# .NET observability with OpenTelemetry
+
+When you run a distributed application, you want to know how well the app is performing and to detect potential problems before they become larger. One way that you can do this is by monitoring specific instrumentation emitted by the application and looking for changes in metrics and logs.
+
+## What is observability
+
+Observability in the context of a distributed system is the ability to monitor and analyze telemetry about the state of each component, to be able to observe changes in performance, and to diagnose why those changes occur. Unlike debugging, which is invasive and can affect the operation of the application, observability is intended to be transparent to the primary operation and have a small enough performance impact that it can be used continuously.
```

Comment:
Very common I've seen "observabiity = monitoring, diagnostics, analytics".
For example:
* Monitor the CPU metrics to get an overall sense of resource utilization.
* Analyze the usage pattern to better forecast the business growth or capacity planning.
* Diagnose a particular distributed trace to see why it is taking minutes to finish.

Reply:
@samsp-msft You've brought up a topic which is very interesting to me (I used to work on debuggers) - I have a gut feeling that debuggers can be improved to be non-invasive for most of the cases and that should be the direction, happy to chat more offline.